### ISSUE TITLE
Liferea always start in wide view on Fedora 27 beta

### ISSUE THREAD

**Issue by medeotl** [lwindolf/liferea#554]
In fedora 27 beta and liferea 1.12-rc3 liferea always start in wide view. Changing to normal view have no effect until clicking on a feed. If you close it in normal view and reopen it, it will be displayed in (empty) wide view again.
![screenshot from 2017-11-06 20-14-04](https://user-images.githubusercontent.com/1864684/32459162-10cf8e1a-c32f-11e7-9a4b-1b91c152d490.png)

**Comment by da2x** [comment]
If you move your config and user data directories to a new folder called lifereabackup, does the problem still happen?

**Comment by medeotl** [comment]
[medeo@localhost ~]$ mv /home/medeo/.cache/liferea/ /home/medeo/.cache/lifereaBACKUP
[medeo@localhost ~]$ mv /home/medeo/.config/liferea/ /home/medeo/.config/lifereaBACKUP
[medeo@localhost ~]$ mv /home/medeo/.local/share/liferea/ /home/medeo/.local/share/lifereaBACKUP

yes, it still behave the same.

**Comment by lwindolf** [comment]
@medeotl Please retest using git master (or 1.12.1) which includes the fix of #504

**Comment by feuerbach** [comment]
I tested this with the current master (a0a7464) and can confirm that the bug still exists. Interestingly, I am also on Fedora 27 — could this be something in Fedora's version of gtk or some other library?

**Comment by lwindolf** [comment]
Ah... I think I now understand. When Liferea starts no feed is selected. So no feed-specific property concerning the view mode can be persisted. This is why the default setting is used which is the wide-mode. So no problem in the code. I'm relieved :-)

Of course one might want to think about if this is user-friendly. Earlier releases had a start screen, which we dropped to show the previously selected feed. But then we had to drop this feature again, because of severe problems. So we ended up loading nothing...
